zcash_client_sqlite/src/scan.rs

```diff
@@ -57,6 +57,38 @@ struct WitnessRow {
 pub fn scan_cached_blocks<P: AsRef<Path>, Q: AsRef<Path>>(
     db_cache: P,
     db_data: Q,
+) -> Result<(), Error> {
+    scan_cached_block_batch(db_cache, db_data, std::i32::MAX)
```

Comment:
```suggestion
    scan_cached_blocks_incremental(db_cache, db_data, std::u32::MAX)
```

Reply:
Stuck with one function approach and original name.

Addressed in ab100d3b35e2130e0b81631cd20585cde2ef9f75